### DISCUSSION TITLE
Onstep

### DISCUSSION THREAD

**Discussion by rtoquato** [Stellarium/stellarium#3789] [Q&A]
Please can some one give some ideias or write how to establish conenction between Onstep V4Pro to Stellarium by usb port?

Thank's

**Comment by gzotti** [comment]
- Read the chapter about telescope support in the User Guide. 
- Check the manual of your device what kind of communication protocol is supported. 
- Act accordingly.

**Comment by sivom** [comment]
@rtoquato  
you can follow this thread I created with more details
https://onstep.groups.io/g/main/topic/106778552#msg59793

1. Install ch34x linux driver
2. https://github.com/juliagoda/CH341SER
3. If you want to use Stellarium, you need to run indiserver, then in stellarium choose indi.
4. I would recommend kstars as it has built in indi and ekos support panel which gives more details on the connection and you will know more in detail what is going on with the driver and the scope.
have fun.
